src/lib/checkers.py

```diff
@@ -35,7 +36,11 @@
 class Checker:
     CHECKER_DATA_TYPE: t.Optional[str] = None
     CHECKER_DATA_SCHEMA: t.Dict[str, t.Any]
-    SUPPORTED_DATA_CLASSES: t.List[t.Type[ExternalBase]] = [ExternalData]
+    SUPPORTED_DATA_CLASSES: t.List[t.Type[ExternalBase]] = [
+        ExternalData,
+        ExternalGitRepo,
```

Comment:
Not sure if this is correct, it was necessary to pass this down below:

https://github.com/flathub/flatpak-external-data-checker/blob/4de54724f00e001df0986a3d01e5575abff1384a/src/lib/checkers.py#L56-L58

I assume this is since my tests happened to be git sources. If it's preferable I can simply move them to html sources or something, it shouldn't matter match.

Though I was curious, why shouldn't this assertion allow a git source through? As far as I can tell, the way my git source tests work using this code makes sense (i.e. this code is operating sanely with git sources despite the original assertion disallowing them)).

Reply:
It is incorrect. This `SUPPORTED_DATA_CLASSES` is the default, you shouldn't change it. Instead, you're supposed to redefine it on child classes.

Reply:
Makes sense, fixed